src/utils/parser/tests/fixtures/expression.js

```diff
@@ -1,6 +1,7 @@
 function expr() {
   const obj = { a: { b: 2 } };
   const obj2 = { c: { b: 3 } };
-  const foo = obj2.c.b;
+  const foo = obj2.c.bazzz;
```

Comment:
hmm maybe this could have a better naming, so if someone goes hunting here it will be clearer what is happening with this expression?